app/helpers/beer_helper.rb

```diff
@@ -0,0 +1,5 @@
+module BeerHelper
+  def beer_owner
```

Comment:
`def beer_owner?` is the idiomatic way to write this method's name if it returns true or false.

Is there a reason you chose a helper as opposed to having an instance method in the beer model that checks for this?